### PR TITLE
Add dev environments to AWS

### PR DESCRIPTION
**Pull request by dincho**
completes https://www.pivotaltracker.com/story/show/160448926

after https://github.com/aeternity/infrastructure/pull/109, https://github.com/aeternity/infrastructure/pull/110

Successful deploy: https://circleci.com/gh/aeternity/epoch/33125 (note that openstack is off due to https://github.com/aeternity/infrastructure/pull/112, thus only AWS nodes has been deployed).

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -88,3 +94,39 @@ module "aws_deploy-us-west-2" {
     aws = "aws.us-west-2"
   }
 }
+
+module "aws_deploy-dev1-eu-west-2" {
```
Comment (lucafavatella) [inline terraform/main.tf]: I see correspondent UAT section has no env in the name so may cause confusion.

**Comment by lucafavatella** [comment]
If this is multistage solution of https://www.pivotaltracker.com/n/projects/2152594/stories/160448926 please identify steps, and we need to clarify timing interaction with testnet deployment (and associated development testing needs - so dev1/dev2 downtime may be relevant).

**Comment by dincho** [comment]
@lucafavatella All the OS environments will be removed in scope of https://www.pivotaltracker.com/story/show/160761831 after this and the other related stories are merged.

I'll update datadog links once this is merged, because I don't know the IP addresses in advance.

In reagards to CircleCI, if you meant to remove [the setup of dev environments](https://github.com/aeternity/infrastructure/blob/master/.circleci/config.yml#L136-L152) I plan to do in https://www.pivotaltracker.com/story/show/160761831


To summarise: I plan to create couple more PR's that **add** integration and fast_integration environments to AWS. No dawn-time of the environments is expected, just growing the node count. After that we'll kill the OS environments (dev1, dev2, integration, fast_integration).

**Comment by dincho** [comment]
@lucafavatella I fixed missing job dependancy 802b76e

File: terraform/main.tf (hunk 1 above)
Comment (dincho) [reply terraform/main.tf]: Yes, it may need separate manual migration, thus not included here.
